docs/features/jsonSchema.md

```diff
@@ -19,32 +19,61 @@ The documentation below describes the new implementation.
 The new JSON Schema is structured to validate the YAML files that are loaded via JCasC.
 The structure and validation of the Schema is done based on the user-installed plugins.
 
-=== Version
+ ***Version***
 
 The Schema uses JSON draft v07. 
 http://json-schema.org/draft-07/schema#
 
-=== How to use
+***Using JSON Schema***
```

Comment:
Ditto for all the `***` changes

Reply:
I changed all the sub-heading with italic texts to simple bold texts.
Could you please check and let me know?